git_machete/cmd.py

```diff
@@ -10,147 +10,23 @@
 import os
 import re
 import shutil
-import subprocess
 import sys
 import textwrap
 
+from git_machete import utils
+from git_machete.utils import RED, YELLOW, ORANGE, GREEN, DIM, BOLD, ENDC, dim, pretty_choices, bold, colored, debug, fmt, underline
+from git_machete.contexts import CommandLineContext
+from git_machete.exceptions import MacheteException, StopTraversal
+from git_machete.docs import short_docs, long_docs
+from git_machete.git_operations import GitContext, REFLOG_ENTRY
+from git_machete import git_operations
```

Comment:
Given how often this module is referenced... maybe just `git`? or will clash with sth else this way?

Reply:
Okay, probably this comment will no longer be relevant once `GitContext` is extracted (since you'll rather use `GitContext` instance rather than reference the module directly)